lib/perl/centreon/script/centcore.pm

```diff
@@ -775,7 +775,7 @@ sub sendExportFile($){
     # Send data with rSync
     $self->{logger}->writeLogInfo('Export files on poller "' . $server_info->{name} . '" (' . $id . ')');
 
-    $cmd = "$self->{rsync} -ra --port=$port $origin $dest 2>&1";
+    $cmd = "$self->{rsync} -ra -e '$self->{ssh} -o port=$port' $origin $dest 2>&1";
```

Comment:
The man of rsync indicates:           --port=PORT             specify double-colon alternate port number
I don't understand your change

Reply:
Yes it's a little bit tricky.
The `--port` option of rsync is used to specify the rsync listening port, when rsync is used in daemon mode (without SSH tunnel).
But here, as we use SSH as the transport tunnel, we have to indicate the destination port to SSH itself.
Thus, we have to use `-e` rsync option to tune the SSH call, simply giving it `-o port=...`.